RedBugs/src/main/java/Data/InitData/WriteReadJson.java

```diff
@@ -0,0 +1,94 @@
+package Data.InitData;
+
+import Data.Route;
+import Data.Stop;
+import Data.Vehicle;
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.List;
+
+public class WriteReadJson {
+
+    public static void main(String[] args) throws Exception {
+        InitData.initData();
+        writeToJsonStops(InitData.stops, "Stops.json");
+        writeToJsonVehicles(InitData.vehicles, "Vehicles.json");
+        writeToJsonRoutes(InitData.routes, "Routes.json");
+
+        readFromJsonStops("Stops.json").stream().forEach(stop -> System.out.println(stop));
+    }
+
+    public static void writeToJsonStops(ArrayList<Stop> arrayList, String fileName) throws Exception {
+
+        ObjectMapper objectMapper = new ObjectMapper();
+        String arrayListAsSting = objectMapper.writerWithDefaultPrettyPrinter().writeValueAsString(arrayList);
+
+        Path path = Paths.get(fileName);
+        Files.write(path, arrayListAsSting.getBytes());
+
+        path.toFile().deleteOnExit();
+    }
+
+    public static void writeToJsonVehicles(ArrayList<Vehicle> objectArrayList, String fileName) throws Exception {
```

Comment:
writeToJson miał być generyczny, tzn przyjmować Object i używając ifa z instanceof parsować na odpowiedni model, a ja tu widzę praktycznie 3 takie same metody write. To samo tyczy się readJson